### ISSUE TITLE
Don't directly print arrays > 32 elements in Debug impls in -sys crates

### ISSUE THREAD

**Issue by sdroege** [gtk-rs/gir#516]
On gstreamer-sys, the following diff is currently generated

```diff
diff --git a/gstreamer-audio-sys/src/lib.rs b/gstreamer-audio-sys/src/lib.rs
index c579643..76f13b0 100644
--- a/gstreamer-audio-sys/src/lib.rs
+++ b/gstreamer-audio-sys/src/lib.rs
@@ -910,7 +910,7 @@ impl ::std::fmt::Debug for GstAudioRingBuffer {
          .field("callback", &self.callback)
          .field("cb_data", &self.cb_data)
          .field("need_reorder", &self.need_reorder)
-         .field("channel_reorder_map", &format!("{:?}", &self.channel_reorder_map as *const _))
+         .field("channel_reorder_map", &self.channel_reorder_map)
          .field("flushing", &self.flushing)
          .field("may_start", &self.may_start)
          .field("active", &self.active)
```

`channel_reorder_map` is defined as
```rust
pub channel_reorder_map: [c_int; 64],
```

This fails to compile because all the traits are only implemented for arrays <= 32 elements

```
error[E0277]: the trait bound `[i32; 64]: bitflags::<unnamed>::fmt::Debug` is not satisfied
   --> gstreamer-audio-sys/src/lib.rs:913:40
    |
913 |          .field("channel_reorder_map", &self.channel_reorder_map)
    |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^ `[i32; 64]` cannot be formatted using `:?`; if it is defined in your crate, add `#[derive(Debug)]` or manually implement it
    |
    = help: the trait `bitflags::<unnamed>::fmt::Debug` is not implemented for `[i32; 64]`
    = note: required for the cast to the object type `bitflags::<unnamed>::fmt::Debug`
```

**Comment by tmiasko** [comment]
This should be fixed now as part of #527. Let me know if you still have an issue with this.

**Comment by GuillaumeGomez** [comment]
Fixed by #527.
